src/helpers.js

```diff
@@ -80,8 +80,27 @@ var insertTask = function(namespace, input, options) {
   });
 };
 
+var getModelData = function({query, limit, continuation, Model}, callback) {
+  return Model.query(query, {
+    limit,
+    continuation,
+  }).then(function(data) {
+    var retval = {};
+    retval.tasks = data.entries.map(function(task) {
```

Comment:
However, you've called it with namespaces as well, and those need to be attached at `retval.namespaces`.  So there are failing tests.  Running `DEBUG=* mocha -b test/api_test.js` (the DEBUG is necessary because API error messages only appear there) gives

```
  api:errors Error occurred handling: /namespaces/, err: Error: Output schema validation error: 
Schema Validation Failed!
Rejecting Schema: http://schemas.taskcluster.net/index/v1/list-namespaces-response.json#
Errors:
  * data should NOT have additional properties
  * data should have required property 'namespaces', as JSON: {"schema":"http://schemas.taskcluster.net/index/v1/list-namespaces-response.json#","url":"/namespaces/","payload":{"tasks":[{"namespace":"CsGbOLrnRxiT7wZcYOU-xg","name":"CsGbOLrnRxiT7wZcYOU-xg","expires":"2018-02-21T00:00:00.000Z"}]}}, incidentId: 3e21a949-eb93-4079-b08c-fecd90682419 Error: Output schema validation error: 
Schema Validation Failed!
Rejecting Schema: http://schemas.taskcluster.net/index/v1/list-namespaces-response.json#
Errors:
  * data should NOT have additional properties
  * data should have required property 'namespaces'
    at ServerResponse.res.reply (/home/dustin/p/taskcluster-index/node_modules/taskcluster-lib-api/lib/api.js:172:22)
    at helpers.getModelData (/home/dustin/p/taskcluster-index/src/api.js:220:100)
    at /home/dustin/p/taskcluster-index/src/helpers.js:95:12
    at <anonymous>
    at process._tickDomainCallback (internal/process/next_tick.js:228:7) +2s
```

Reply:
changed the signature of the helper method to accept a key, that forms the key of the return value.